test/test_compile.py

```diff
@@ -0,0 +1,28 @@
+import torch 
```

Comment:
nit: it seems weird to add a dedicated test file for this when we already have `test/dynamo`